docs/en/latest/concepts/apisix_upstream.md

```diff
@@ -193,3 +193,33 @@ In the meanwhile, the ApisixUpstream `foo` sets `http` scheme for port `7000` an
 (all ports are the service port). But both ports shares the load balancer configuration.
 
 `PortLevelSettings` is not mandatory if the service only exposes one port but is useful when multiple ports are defined.
+
+### Http Host Rewrite Policy
```

Comment:
Add an empty line after this title.

Reply:
thanks for pointing out, rerun markdownlint and gofmt locally to fix the format issue.